src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2006,6 +2007,8 @@ private function registerMailerConfiguration(array $config, ContainerBuilder $co
         $envelopeListener = $container->getDefinition('mailer.envelope_listener');
         $envelopeListener->setArgument(0, $sender);
         $envelopeListener->setArgument(1, $recipients);
+
+        NotificationEmail::setDefaultTemplates($config['notification']['templates'][0] ?? null, $config['notification']['templates'][1] ?? null);
```

Comment:
This looks wrong to me. This static call will be done only at compile time, and so won't be applied for any request using a cached container. This needs to be moved to `boot()` of the bundle (or dropped)

Reply:
Oops, fixed.